handlers/graph.go

```diff
@@ -84,9 +84,11 @@ func graphNamespaces(o options.Options, client *prometheus.Client) *[]*tree.Serv
 	log.Debugf("Build graph for [%v] namespaces [%s]", len(o.Namespaces), o.Namespaces)
 
 	roots := []*tree.ServiceNode{}
+	allNamespaces := (len(o.Namespaces) > 1)
```

Comment:
Multiple namespaces does not imply 'all' namespaces.   If you need to pass a flag it should be renamed to 'multipleNamespaces' or something like that.  To be honest. it may be nicer to avoid special-case code while building the trees and instead update the finished trees in another pass, it will keep the logic contained.